modman.php

```diff
@@ -587,10 +589,34 @@ public function doDeploy($bForce = false) {
 			}
 			if (!is_link($oLine->getSymlink())) {
 				echo ' Applied: ' . $oLine->getSymlink() . ' ' . $sFullTarget . PHP_EOL;
-				symlink(
-					$sFullTarget,
-					$oLine->getSymlink()
-				);
+
+				if ($bCopy) {
+					if (is_dir($sFullTarget)) {
```

Comment:
Well it might be not recursive, but can't we start iteration right from the beginning?